docs/Getting Started/openSUSE/openSUSE Leap Root on ZFS.rst

```diff
@@ -1074,6 +1091,16 @@ available. There is `a bug report upstream
    compression algorithm can reduce I/O. The exception is all-zero pages,
    which are dropped by ZFS; but some form of compression has to be enabled
    to get this behavior.
+   
+   **Hints:** If you saw warning about uncorrect volblocksize, recreate swap device:
```

Comment:
```suggestion
   **Hints:** If you've got a warning about incorrect volblocksize, recreate swap device:
```

Reply:
Hm, but what's the case when user can get such problems? `getconf PAGESIZE` should work.